content/_code-samples/tx-serialization/js/main.js

```diff
@@ -0,0 +1,36 @@
+const { Client, encode } = require("xrpl");
+
+const payload = require("./json_input.json");
+
+const main = async () => {
+    try {
+        console.log(`Converting your transaction...\n`);
+
+        // connect to a public server
+        const client = new Client("wss://xrplcluster.com/");
```

Comment:
Instead of pointing to a mainnet node, it'd be better to point at a test network so that people who use this code snippet can play around with the values themselves.